src/customTypes.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
```

Comment:
Suggest to rename as `interfaces.ts`  or `types.ts`

Reply:
I think it would be good to leave the interface definition on mapping handler, would be more straightforward when reading the code.